cirq-google/cirq_google/engine/engine_client.py

```diff
@@ -88,10 +88,20 @@ def submit(self, func: Callable[..., Awaitable[_R]], *args, **kw) -> duet.Awaita
 
 
 class EngineClient:
-    """Client for the Quantum Engine API that deals with the engine protos and
-    the gRPC client but not cirq protos or objects. All users are likely better
-    served by using the Engine, EngineProgram, EngineJob, EngineProcessor, and
-    Calibration objects instead of using this directly.
+    """Client for the Quantum Engine API handling protos and gRPC client.
+
+    Thie is the client for the Quantum Engine API that deals with the engine protos
+    and the gRPC client but not cirq protos or objects. All users are likely better
+    served by using the `Engine`, `EngineProgram`, `EngineJob`, `EngineProcessor`, and
+    `Calibration` objects instead of using this directly.
+
+    Args:
```

Comment:
does this render correctly on doc site?  I thought you should keep the args in the init but put a description on the class for the best redering?

Reply:
Changed to this style.